package.json

```diff
@@ -173,6 +173,7 @@
     "postinstall": "bower install",
     "pretest": "yarn run lint",
     "dev": "yarn install && gulp dev",
+    "prod": "yarn install && NODE_ENV=production gulp build",
```

Comment:
I don’t think there’s a need for the `yarn install` here—that’s already a step in Travis, and best to keep separate things separate imo (the `dev` thing is to make it more convenient for humans running the dev server, but no humans to inconvenience in production builds)